src/django_components/html.py

```diff
@@ -0,0 +1,98 @@
+import re
```

Comment:
Helpers for working with the `selectolax` HTML objects. In the future I want to break down the current `utils.py` file and instead make it into a directory, so this file could be moved to `utils/html.py`.